### PR TITLE
chore: reformat info.xml

### PR DESCRIPTION
**Pull request by st3iny**
My OCD strikes again :D

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -15,10 +15,10 @@
 	</types>
 	<category>office</category>
 	<category>organization</category>
-    <bugs>https://github.com/nextcloud/calendar_resource_management</bugs>
-    <dependencies>
+	<bugs>https://github.com/nextcloud/calendar_resource_management</bugs>
+	<dependencies>
 		<php min-version="8.0" max-version="8.3" />
-		<nextcloud min-version="26" max-version="29"/>
+		<nextcloud min-version="26" max-version="30"/>
```
Comment (ChristophWurst) [inline appinfo/info.xml:21]: sneaky

Comment (st3iny) [reply appinfo/info.xml:21]: :see_no_evil:

Comment (st3iny) [reply appinfo/info.xml]: Fixed

**Comment by st3iny** [comment]
https://github.com/nextcloud/calendar_resource_management/pull/123/files?w=1

**Review by ChristophWurst — APPROVED** [review]
☯️

**Review by miaulalala — APPROVED** [review]

**Pull request merged**